### DISCUSSION TITLE
Video uploads stuck on PROCESSING…

### DISCUSSION THREAD

**Discussion by ldesousa** [mastodon/mastodon#31311] [Q&A]
I have been trying to upload a video as an attachment to a post since yesterday. Every time the upload seems to succeed, but then the post editing form remains on the "PROCESSING…" step forever. The only restriction I am aware to video upload is the size limit, 99 MB, which this video is well under. Could I be doing something wrong?

I am on the mastodon.social instance.

**Comment by valpackett** [comment]
this is also reported as an issue:  #32899
